handbook/security-policies.md

```diff
@@ -43,4 +43,19 @@ Fleet requires all workforce members to comply with the following acceptable use
 
 13. The use of Fleet company accounts on "shared" computers, such as hotel kiosk systems, is strictly prohibited.
 
+### Risk management policy
+*Created from [JupiterOne/security-policy-templates](https://github.com/JupiterOne/security-policy-templates). [CC BY-SA 4 license](https://creativecommons.org/licenses/by-sa/4.0/)*
```

Comment:
@GuillaumeRoss Maybe consider writing this using `> _Created from…_`.  Might look better when styled.  If you do this, I would do it in all the other places for consistency.  This is not a blocker to merging this.

Reply:
I took a note to do that for all of them once I am done. Thanks!

Reply:
https://github.com/fleetdm/fleet/pull/6625